mobile/lib/features/cqrs/command_result.dart

```diff
@@ -0,0 +1,14 @@
+abstract class CommandResult {}
+
+class SuccessCommandResult extends CommandResult {
```

Comment:
I assume every command should return id?

Reply:
No, if validation fails, command won't run thus no ID will be generated.